### PR TITLE
Inline documentation for iteritems, dropna, drop, get

### PR DESCRIPTION
**Pull request by shril**
Inline documentation for drop function in frame.py file.

### CODE REVIEW

**Comment by shril** [comment]
Hi @rxin , have a look at this.
I am working on the inline docs of other functions in frame.py file.

Happy to address any issues regarding my changes.

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -881,8 +881,47 @@ def unique(self):
         sdf = self._sdf
         return DataFrame(spark.DataFrame(sdf._jdf.distinct(), sdf.sql_ctx), self._metadata.copy())
 
-    @derived_from(pd.DataFrame)
     def drop(self, labels, axis=0, errors='raise'):
```
Comment (rxin) [inline databricks/koalas/frame.py]: cc @thunterdb why do we have "errors" parameter that seem unused?

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -881,8 +881,47 @@ def unique(self):
...
+        Drop specified labels from columns.
+
+        Remove columns by specifying label names and axis=1.
+        Removing rows is yet to be implemented.
+
+        Parameters
+        ----------
+        labels : single label or list-like
+            Column labels to drop.
+        axis : {0 or 'index', 1 or 'columns'}, default 0
```
Comment (rxin) [inline databricks/koalas/frame.py]: say something about only axis = 1 is supported at the moment.

**Comment by rxin** [comment]
This looks great. For the next pull request, feel free to do more than one function.

**Comment by shril** [comment]
@rxin I'll push the documentation of other functions as well in this PR itself.
I little caught up.

File: databricks/koalas/frame.py (hunk 2 above)
Comment (shril) [reply databricks/koalas/frame.py]: Added the information.

**Comment by shril** [comment]
Hi @rxin , I have added documentation for several functions in this PR.

I was keeping a track of documentation of functions left in frame.py. Issue #152 
Unchecked ones are left.

- [x] to_html
- [x] isnull
- [x] notnull
- [x] toPandas
- [ ] assign
- [x] dropna
- [x] drop
- [x] get
- [ ] pipe
- [x] iteritems

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -163,8 +163,44 @@ def corr(self, method='pearson'):
...
 
-    @derived_from(pd.DataFrame)
     def iteritems(self):
+        """
+        Iterator over (column name, Series) pairs.
+
+        Iterates over the DataFrame columns, returning a tuple with
+        the column name and the content as a Series.
+
+        Yields
```
Comment (HyukjinKwon) [inline databricks/koalas/frame.py]: Let's make this API yield or fix the doc.

File: databricks/koalas/frame.py [hunk 4]
```diff
@@ -691,8 +727,88 @@ def loc(self):
...
         return DataFrame(self._sdf, self._metadata.copy())
 
-    @derived_from(pd.DataFrame)
     def dropna(self, axis=0, how='any', thresh=None, subset=None, inplace=False):
+        """
+        Remove missing values.
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns'}, default 0
```
Comment (HyukjinKwon) [inline databricks/koalas/frame.py]: I think we don;t support `columns` yet. Let's take out.

File: databricks/koalas/frame.py (hunk 1 above)
Comment (HyukjinKwon) [reply databricks/koalas/frame.py]: Seems like just wanted to match the signature (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop.html#pandas.DataFrame.drop) Let's remove.

File: databricks/koalas/frame.py (hunk 2 above)
Comment (HyukjinKwon) [reply databricks/koalas/frame.py]: I think we should fix this doc.

File: databricks/koalas/frame.py (hunk 3 above)
Comment (shril) [reply databricks/koalas/frame.py]: I am fixing the doc for now.

File: databricks/koalas/frame.py (hunk 4 above)
Comment (shril) [reply databricks/koalas/frame.py]: Doing it.

File: databricks/koalas/frame.py (hunk 1 above)
Comment (shril) [reply databricks/koalas/frame.py]: Removing it.

**Comment by shril** [comment]
Hi @HyukjinKwon,

- Changed the docs for `dropna`, which only supports axis=0 at the moment.
- Changed the docs for `drop`, which only supports axis=1 at the moment.
- Changed from yields to returns in the documentation for `iteritems`.
- Removed error parameter from the `drop` function.

**Comment by shril** [comment]
Hi @HyukjinKwon, I think my PR contains a large number of commits.
Should I fork another branch from master and transfer the changes to a single commit?

**Comment by rxin** [comment]
It's ok when we merge into master we squash all commits into a single one.

File: databricks/koalas/frame.py [hunk 5]
```diff
@@ -691,8 +727,86 @@ def loc(self):
...
+        Returns
+        -------
+        DataFrame
+            DataFrame with NA entries dropped from it.
+
+        See Also
+        --------
+        DataFrame.drop : Drop specified labels from columns.
+        DataFrame.isnull: Indicate missing values.
+        DataFrame.notnull : Indicate existing (non-missing) values.
```
Comment (garawalid) [inline databricks/koalas/frame.py]: We don't have `notnull()`.
You can replace it by `notna()`.

Comment (rxin) [reply databricks/koalas/frame.py]: We do have notnull, don't we? https://github.com/databricks/koalas/blob/master/databricks/koalas/frame.py#L535

**Comment by rxin** [comment]
Thanks. I've merged this in master.

File: databricks/koalas/frame.py (hunk 5 above)
Comment (shril) [reply databricks/koalas/frame.py]: Line 571 is `notnull()` in frame.py.
Please correct me if I am making any mistake here.

**Comment by HyukjinKwon** [comment]
Nice doc change. Thanks, @shril

File: databricks/koalas/frame.py (hunk 5 above)
Comment (garawalid) [reply databricks/koalas/frame.py]: I saw the documentation. You re right

**Pull request merged**